OpenEdXMobile/res/values/strings.xml

```diff
@@ -584,6 +584,8 @@
     <string name="profile_incomplete_about_me">No “About me” info provided.</string>
     <!-- Window title for edit profile screen -->
     <string name="edit_user_profile_title">Edit profile</string>
+    <!-- Content description for edit profile username -->
+    <string name="edit_user_username_content_description">Your username is {username}</string>
```

Comment:
"_content" is meaningless here, as the preceding `edit_user_username` already describes the content, so it would be better to just add a `_description` suffix. This applies to the new `profile_change_photo` description as well.